wiki/Medals/Unlock_requirements/Hush-hush/en.md

```diff
@@ -0,0 +1,8 @@
+# Unlock requirements for Hush-Hush medals
+
+*Note: While some conditions for Hush-Hush medals are known, their solutions are intentionally omitted from this wiki to help preserve their secrecy.*
+
+- If a medal does not appear in a gamemode's profile, this medal cannot be obtained in that particular gamemode. If said medal is shown in all gamemodes' profiles, in most cases it means you can obtain it in any gamemode, but there are exceptions due to various reasons, such as star rating changes.

```

Comment:
```suggestion
- A medal only appears in a [game mode](/wiki/Game_mode)'s profile it can be obtained in. However, there are exceptions due to various reasons, such as star rating changes.
```

Reply:
```suggestion
- A medal only appears in the profile of [gamemodes](/wiki/Game_mode) it can be obtained in. However, there are exceptions due to various reasons, such as star rating changes.

```